tests/unit/NetworkTest.js

```diff
@@ -132,22 +132,8 @@ test('consecutive API calls eventually succeed when authToken is expired', () =>
     const TEST_CHAT_LIST = [1, 2, 3];
 
     let chatList;
-    Onyx.connect({
-        key: 'test_chatList',
-        callback: val => chatList = val,
-    });
-
     let account;
-    Onyx.connect({
-        key: 'test_account',
-        callback: val => account = val,
-    });
-
     let personalDetailsList;
-    Onyx.connect({
-        key: 'test_personalDetailsList',
-        callback: val => personalDetailsList = val,
-    });
```

Comment:
It seems unnecessary to merge results to Onyx only to update the local variables here when we can update them directly